src/qibolab/dummy/parameters.json

```
@@ -678,6 +683,24 @@
                 "mixer_readout_phi": 0.0
             }
         },
+        "two_qubit":{
+            "0-2": {
+                "twoq_fidelity": 0.0,
```

```suggestion
                "gate_fidelity": 0.0,
```
Since it is already in `QubitPair`